src/main/java/org/kohsuke/github/GitHubRequest.java

```diff
@@ -437,6 +437,10 @@ public B withPreview(String name) {
             return withHeader("Accept", name);
         }
 
+        public B withPreview(Previews preview) {
+            return withPreview(preview.mediaType());
+        }
+
```

Comment:
This overload of the `withPreview` method was defined to prevent us from having to update every usage of the method after `Previews` was converted to an enum